src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/constructor/json/ConstructYamlBinary.kt

```diff
@@ -25,7 +25,11 @@ class ConstructYamlBinary : ConstructScalar() {
         // Ignore white spaces for base64 encoded scalar
         // TODO decodeBase64() doesn't seem to require removing whitespace - perhaps this can be removed?
         //      I'm leaving it in because I'm not confident about edge case test coverage for whitespaces.
-        val noWhiteSpaces = constructScalar(node).replace("\\s".toRegex(), "")
+        val noWhiteSpaces = constructScalar(node).replace(SPACES_PATTERN, "")
         return noWhiteSpaces.decodeBase64()?.toByteArray() ?: byteArrayOf()
     }
+
+    companion object {
```

Comment:
suggestion: I think if we add `private` modifier for companion object we should not get any changes in the API file

Reply:
I just followed your request to make it like the other cases where it is done like that. :-D
I just made it a `private val` in the Kotlin file now, unless you really prefer a private companion object.

Reply:
I'm not requesting anything) I just left some suggestions - everyone is free to disagree with them or ignore them completely) If something is an issue that doesn't let the PR go forward, in my opinion, it will be clearly indicated.

Regarding the change itself: I don't think we want to change a public API just because of introducing a constant (that is what just happened). Because of that I suggest we try to make the companion object private (or maybe internal) to avoid changes in public API

Reply:
Requesting, suggesting, doesn't make too much difference in this context. :-D
For me a collaborator doing a PR review and suggesting changes is requesting them to be done. :-D